wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/SubsystemBase.java

```diff
@@ -14,14 +14,14 @@
  *
  * <p>This class is provided by the NewCommands VendorDep
  */
-public abstract class SubsystemBase implements Subsystem, Sendable {
+public abstract class SubsystemBase implements Resource, Sendable {
```

Comment:
Is not renaming SubsystemBase intentional? It seems like that's what most people would use in their code, so I'm not seeing the benefit of the name change.

Reply:
On the fence about renaming `SubsystemBase`, since the `Sendable` stuff doesn't necessarily make sense to scope at the `Resource` level.